src/test/java/uk/gov/hmcts/reform/lrdapi/controllers/LrdCourtVenueControllerTest.java

```diff
@@ -154,4 +175,10 @@ void testGetCourtVenuesBySearchStringWithInvalidCourtTypeIdWithComma() {
         assertThrows(InvalidRequestException.class, () ->
             lrdCourtVenueController.retrieveCourtVenuesBySearchString("ABC", ",1,2,", null, null, null, null));
     }
+
+    @Test
+    void testGetCourtVenuesBySearchStringWithConsecutiveSpecialCharacters() {
```

Comment:
can you add other search strings with only repeated special chars like 3 underscores and likewise other all allowed special chars, please ?

Reply:
sure, I'll add them.

Reply:
addressed here [7a3ac72](https://github.com/hmcts/rd-location-ref-api/pull/473/commits/7a3ac72a8c9d5afffdb33a70158fc51a14a52465)